src/redux/ConfigureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import BooksReducer from './books/books';
+import { ConfigureStateReducer } from './Categories/categories';
```

Comment:
- [ ] as part of the requirements, kindly create a categories reducer, where :
* in case `action.type` is `CHECK_STATUS` the function returns a string "Under construction".
* if the `action.type` is different, it returns the state with no changes.
kindly fix that , and make sure ur action types follows the ducks patterns 🙏